client/src/components/stripe/CheckoutForm.js

```diff
@@ -0,0 +1,111 @@
+import React, { useState,useContext,useEffect } from 'react';
+import { useStripe, useElements, CardElement } from '@stripe/react-stripe-js';
+import CardSection from './CardSection';
+import Button from '@material-ui/core/Button';
+import Grid from '@material-ui/core/Grid';
+import { AlertContext } from '../../globals/AlertContext';
+import { UserContext } from '../../globals/UserContext';
+
+
+function stripeCheckoutHandler(result,email,alertContext) {
+
+    if (result.error) {
+        alertContext.setAlertStatus({
+            isOpen:true,
+            message:result.error.message,
+            type:"error"
+        })
+    } else {
+        fetch('/create-subscription', {
+            method: 'POST',
+            headers: { 'Content-Type': 'application/json' },
+            credentials: "include",
+            body: JSON.stringify({
+                priceId: 'price_1IY2oXFSBPZAgVJdLwijjbRb',
+                payment_method_id: result.paymentMethod.id,
+                email: email
+            })
+        }).then(function (result) {
+            // Handle server response (see Step 4)
+            result.json().then(function (json) {
+                //handleServerResponse(json);
+            })
+        });
+    }
+}
+
+export default function CheckoutForm() {
+    const stripe = useStripe();
+    const elements = useElements();
+    const alertContext = useContext(AlertContext) 
+    const userContext = useContext(UserContext)
+    const user_id = userContext.userId
+    // const user_id = 10
+    const [email, setEmail] = useState("tofnag23@gmail.com")
+
+    // useEffect( ()=>{
+    //     let status=200
+    //     fetch(`/user/${user_id}/email`, {
+    //         method: "GET",
+    //         headers: {
+    //             "Content-Type": "application/json"
+    //         },
+    //         credentials:"include"
+    //         })
+    //         .then((res) => {
+    //             status = res.status
+    //             if (status < 500)
+    //                 return res.json()
+    //             else throw Error("Server error");
+    //         })
+    //         .then((data) => {
+    //             if (status === 200)
+    //                 setEmail(data.email)
+    //             else throw Error("User email not set");
+    //         })
+    //         .catch(err => {
+    //             alertContext.setAlertStatus({
+    //                 isOpen:true,
+    //                 message:err.message,
+    //                 type:"error"
+    //                 })    
+    //         });
+    // })
+
```

Comment:
delete the comment if we don't need it. And can you upload a screenshot for the upgrade page?

Reply:
here they are:
<img width="1667" alt="Screen Shot 2021-03-29 at 9 59 29 PM" src="https://user-images.githubusercontent.com/32600876/112922301-459b2d80-90da-11eb-879d-7d2e69b5d99e.png">
<img width="1667" alt="Screen Shot 2021-03-29 at 9 59 43 PM" src="https://user-images.githubusercontent.com/32600876/112922303-459b2d80-90da-11eb-8312-2dadc18f42e4.png">